docs/UserGuide.md

```diff
@@ -287,6 +288,30 @@ When the user enters `review` in the command box, this new window will appear. <
 ![UiReview](./images/UiReviewModeWindow.png) <br><br>
 Format: `review`
 
+### Viewing statistics of cards: `stats`
+Shows the statistics of an individual flashcard, or the statistics of all flashcards.
+
+The following statistics are displayed:
+* Number of times the user reviewed the flashcard(s).
+* Number of times the user reviewed the flashcard(s) and got the correct answers.
+* The correct rate of the flashcard(s). i.e The number of correct answer reviews over the number of total reviews.
+* The wrong rate of the flashcard(s). i.e The number of wrong answer reviews over the number of total reviews.
+
+Format: `stats INDEX` <br>
+Note: If a valid INDEX is provided, the statistics of the flashcard identified by the provided index is shown.
+If the INDEX field is omitted, Flashback will display overall statistics for all flashcards in the current list.
+
+Examples:
+
+When user enters `stats 2` in the command box and presses `enter`, the statistics of the <br>
+2nd flashcard in the list is displayed.
+
+![UiReview](./images/UiStatsIndex.png) <br><br>
+
+When user enters `stats` in the command box and presses `enter`, the overall statistics of <br>
+the current flashcard list is displayed.
+
+![UiReview](./images/UiStatsIndex.png) <br><br>
```

Comment:
![image](https://user-images.githubusercontent.com/61836453/112410222-d332f200-8d55-11eb-970d-2ccf484bf368.png)
I'm unable to see the image, you might want to check the link again

Reply:
Fixed. It should show now.